server/api/schema.ts

```diff
@@ -111,6 +111,12 @@ const typeDefs = gql`
     # input FilterOptions {
     #     NOT_AVAILABLE: Boolean
     # }
+    
+    input DonationFormFilterOptions {
+        name: String,
```

Comment:
Could we filter by request group instead of name?

Reply:
I feel like we could have both, they're optional anyways.

Reply:
Let's choose just one. If the user specifies a name and a request group that are different, then we would have to choose between them.

If we go with filtering by name, then we would need to get the request group from the db. whereas if we filter by request group id, we can avoid getting the request group from the db

Also I don't think we're adding any feature that needs us to filter generic donation forms (donation forms without request groups)

Reply:
Done

Reply:
Done